src/tests/ftest/pool/RebuildTests.py

```diff
@@ -73,39 +97,13 @@ def test_simple_rebuild(self):
 
         :avocado: tags=pool,rebuild,rebuildsimple
         """
-
-        # the rebuild tests need to redo this stuff each time so not in setup
-        # as it usually would be
-        setid = self.params.get("setname", '/run/testparams/setnames/')
-        server_group = self.params.get("server_group",
-                                       '/server/',
-                                       'daos_server')
-
-        basepath = os.path.normpath(self.build_paths['PREFIX']  + "/../")
-        tmp = self.build_paths['PREFIX'] + '/tmp'
-
-        self.hostlist = self.params.get("test_machines", '/run/hosts/')
-        hostfile = WriteHostFile.WriteHostFile(self.hostlist, tmp)
-
         try:
-            ServerUtils.runServer(hostfile, server_group, basepath)
-
-            # use the uid/gid of the user running the test, these should
-            # be perfectly valid
-            createuid = os.geteuid()
-            creategid = os.getegid()
-
-            # parameters used in pool create that are in yaml
-            createmode = self.params.get("mode", '/run/testparams/createmode/')
-            createsetid = self.params.get("setname",
-                                          '/run/testparams/createset/')
-            createsize = self.params.get("size", '/run/testparams/createsize/')
 
             # initialize a python pool object then create the underlying
             # daos storage
-            pool = DaosPool(self.Context)
-            pool.create(createmode, createuid, creategid,
-                        createsize, createsetid, None)
+            pool = DaosPool(self.context)
+            pool.create(self.createmode, self.createuid, self.creategid, self.createsize,
```

Comment:
(pylint-line-too-long) Line too long (89/80)

Reply:
no belong to current patch.